include/particles/particle.h

```diff
@@ -329,18 +334,12 @@ class Particle : public ParticleBase<Tdim> {
   Eigen::Matrix<double, 6, 1> strain_rate_;
   //! dstrains
   Eigen::Matrix<double, 6, 1> dstrain_;
-  //! Velocity
-  Eigen::Matrix<double, Tdim, 1> velocity_;
-  //! Displacement
-  Eigen::Matrix<double, Tdim, 1> displacement_;
   //! Particle velocity constraints
   std::map<unsigned, double> particle_velocity_constraints_;
   //! Set traction
   bool set_traction_{false};
   //! Surface Traction (given as a stress; force/area)
   Eigen::Matrix<double, Tdim, 1> traction_;
```

Comment:
Why not put the `traction_` into the `vector_properties_`?

Reply:
As discussed, BC will also be in a future PR.